concrete/views/panels/page/versions.php

```diff
@@ -24,8 +24,14 @@
 			<% } %>
 			<div class="ccm-panel-page-versions-more-info">
 				<p><?=t('Edit by')?> <%-cvAuthorUserName%></p>
-				<% if (cvIsApproved == 1 && cvApprovedDate && cvApproverUserName) { %>
-					<p><?=t('Approved on')?> <%-cvApprovedDate%> <?= t('by'); ?> <%-cvApproverUserName%></p>
+				<% if (cvIsApproved == 1) { %>
+					<% if (cvApprovedDate && cvApproverUserName) { %>
+						<p><?= t('Approved on'); ?> <%-cvApprovedDate%> <?= t('by'); ?> <%-cvApproverUserName%></p>
```

Comment:
This should use tc for By, similar to line 37

Though there should be a PR for changing this file to use the t function correctly as not all languages follow the Scheduled By X for Y/ Approved on X by Y format..

Reply:
I'd prefer a lot to use the whole string `Approved on %1$s by %2$s`.
This can be done with this code:
```
<p><%- <?=json_encode('Approved on %1$s by %2$s') ?>.replace('%1$s', cvApprovedDate).replace('%2$s', cvApproverUserName) %></p>
```

To better understand it, think what the server sends to the client, that is:
```
<p><%- "Approved on %1$s by %2$s".replace('%1$s', cvApprovedDate).replace('%2$s', cvApproverUserName) %></p>
```

See [here](https://jsfiddle.net/fketom4a/) for a working example.

Reply:
Wouldnt this be better? rather than use the JS to do it
```
<p><%- <?= json_encode(t(/*i18n: %1$s is a date approved, %2$s is the approving user*/'Approved on %1$s by %2$s','cvApprovedDate','cvApproverUserName')) ?> %></p>
```
I agree with you @mlocati but as I said its probably better for another PR addressing all of the issues in this file, marvinde is just following the existing format.

Reply:
@deek87 We can't do that:

```php
<p><%- <?= json_encode(t(/*i18n: %1$s is a date approved, %2$s is the approving user*/'Approved on %1$s by %2$s','cvApprovedDate','cvApproverUserName')) ?> %></p>
```
would be sent to the client as
```html
<p><%- "Approved on cvApprovedDate by cvApproverUserName" %></p>
```

As you can see, the template engine would only see a string, so it will print it out as it is ([example](https://jsfiddle.net/uopyw1k5/)).

Reply:
> I agree with you @mlocati but as I said its probably better for another PR addressing all of the issues in this file, marvinde is just following the existing format.

I do agree

Reply:
Oh yeah i forgot about the <% not being able to be passed through